chapters/samplers.txt

```diff
@@ -478,6 +478,51 @@ ifdef::VK_VERSION_1_2[<<features-samplerMirrorClampToEdge,samplerMirrorClampToEd
 endif::VK_VERSION_1_2,VK_KHR_sampler_mirror_clamp_to_edge[]
 --
 
+[open,refpage='VkCompareOp',desc='Comparison operator for depth, stencil, and sampler operations',type='enums']
+--
+_Comparison operators_ compare a _reference_ and a _target_ value, and
+return a true or false value depending on the comparison operator chosen.
```

Comment:
Esthetically, would prefer "test passed" and "test failed" nomenclature rather than raw bools.

Otherwise should replace references to "test" below.